andela-oakinwa/document-management-system
# Feature/147163035/document modal message

#### What does this PR do ?
- Create a delete message before deleting a user or document
- Merge changes to `development` branch.

#### Description of Task to be completed
- Create a delete message before deleting a user or document
- 
#### How should this be manually tested?
- On the Documents page click delete.
- A message would be displayed asking the user to confirm delete action.

#### Any background context you want to provide?
- Delete feature not working on users or documents
- Search is buggy

#### Relevant Pivotal Tracker stories
- #147163035
